src/index.html

```diff
@@ -5,8 +5,223 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- <h1>Catalog</h1> -->
+    <header class="menu">
+      <a class="logo" href="">
+        <img
+        src="./images/logo.png"
+        alt="Logo"
+        >
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="Logo"
        >
```
не забывай про отступы у атрибутов